apps/etterlatte-behandling/src/main/kotlin/behandling/sjekkliste/SjekklisteService.kt

```diff
@@ -44,7 +45,12 @@ class SjekklisteService(
         val items =
             when (behandling.sak.sakType) {
                 SakType.BARNEPENSJON -> defaultSjekklisteItemsBP
-                SakType.OMSTILLINGSSTOENAD -> defaultSjekklisteItemsOMS
+                SakType.OMSTILLINGSSTOENAD -> {
```

Comment:
Kan ikke se at sjekkliste er aktivert for revurderinger ennå (`BehandlingSideMeny.tsx`), så det nye elementet kunne nok fint vært lagt direkte inn i default-lista